tests/functional/config_files/config.py

```diff
@@ -20,6 +20,7 @@
         'redirect_uri': 'REDIRECT_URI',
         'apigee_api_token': 'APIGEE_API_TOKEN',
         'internal_dev_asid': 'INTERNAL_DEV_ASID',
+        'oauth-proxy': 'OAUTH_PROXY'
```

Comment:
See [comment](https://github.com/NHSDigital/personal-demographics-service-api/pull/898/files#r1322778740)

Reply:
Mentioned in SPINEDEM-1680